app/src/main/java/org/mozilla/tv/firefox/ext/Js.kt

```diff
@@ -90,6 +91,18 @@ var _firefoxTV_isPlaybackStateObserverLoaded;
         javaInterface.syncPlaybackState(isVideoPresent, isPlaying, positionSeconds, playbackRate);
     }
 
+    function adjustVideoCSS() {
+        const rawVideoElements = document.getElementsByTagName('video');
+        // Extract elements from HTMLCollection and collect them into an Array
+        const videoElements = new Array(...rawVideoElements);
```

Comment:
nit: `querySelectorAll('video')` will return a `NodeList` that has a `forEach` method on it, invalidating the need to convert the data type. See elsewhere in this file.

If you don't use that, `Array.from(rawVideoElements)`, as a function call, is probably more intuitive for us than its `...` syntactic equivalent here.